<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>perso</class>
 <widget class="QDialog" name="perso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_pic0">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-40</y>
     <width>1051</width>
     <height>891</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../fond3.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_marque">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>19</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Marque</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_matricule">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>19</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>19</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Matricule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouter">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
border:1px solid grey;
background:teal;
color:white;</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_ids">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>19</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>321</width>
     <height>510</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
border:1px solid grey;
background:teal;
color:white;</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ids">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
border:1px solid grey;
background:teal;
color:white;</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>510</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
   <item>
    <property name="text">
     <string>...</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Matricule</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Marque</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>510</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>570</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
border:1px solid grey;
background:teal;
color:white;</string>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>19</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Chercher :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>19</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Trier par:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>570</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 10px;</string>
   </property>
   <item>
    <property name="text">
     <string>...</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Matricule</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Marque</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
border:1px solid grey;
background:teal;
color:white;</string>
   </property>
   <property name="text">
    <string>Trier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline; color:#ff0000;&quot;&gt;PARKING PERSONNELS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:7px;
border:1px solid grey;
background:teal;
color:white;</string>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </widget>
  <zorder>label_pic0</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit_id</zorder>
  <zorder>lineEdit_marque</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_matricule</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_ids</zorder>
  <zorder>tableView</zorder>
  <zorder>lineEdit_ids</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>ajouter</zorder>
  <zorder>comboBox</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButton_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
